### ISSUE TITLE
Implement Discussions

### ISSUE THREAD

**Issue by dkent600** [PrimeDAO/prime-deals-dapp#42]

**Comment by BartuCulha** [comment]
Im not able to reply nor comment to this deal @giladt 

https://prime-deals-staging.vercel.app/deal/open_deals_stream_hash_2

**Comment by curvelabs-dev** [comment]
> Im not able to reply nor comment to this deal @giladt https://prime-deals-staging.vercel.app/deal/open_deals_stream_hash_2 Is there a deal where this address is a lead or representative of: 0x8625F29e4d06D0a3998Ed8C9E45F4b04C7b28D00

There is one test proposal lead given in Notion test instructions

**Comment by BartuCulha** [comment]
- [ ] I should be able to edit my own comments

![DEAL-Dashboard-Prime-Deals (1)](https://user-images.githubusercontent.com/24368665/157450829-25d94db7-5ba2-4543-9ff8-229096c43983.png)

**Comment by BartuCulha** [comment]
- [ ] Maybe it would be better to show < 1m instead of seconds

- [ ] it gets stuck at 1sec, either it should count up or we should display minutes instead of seconds

![DEAL-Dashboard-Prime-Deals (2)](https://user-images.githubusercontent.com/24368665/157450921-aaa3db70-199d-4f9b-bdee-75c1c0c58568.png)

**Comment by giladt** [comment]
> * [ ]  I should be able to edit my own comments
> 
> ![DEAL-Dashboard-Prime-Deals (1)](https://user-images.githubusercontent.com/24368665/157450829-25d94db7-5ba2-4543-9ff8-229096c43983.png)

Edit is not possible ATM, as `theconvo.space` does not support it yet.
Should I remove this button completely for now?
@BartuCulha

**Comment by curvelabs-dev** [comment]
Yes remove the edit button but please create a separate ticket for that.  @giladt

**Comment by curvelabs-dev** [comment]
> * [x]  Maybe it would be better to show < 1m instead of seconds
> * [x]  it gets stuck at 1sec, either it should count up or we should display minutes instead of seconds
> 
> ![DEAL-Dashboard-Prime-Deals (2)](https://user-images.githubusercontent.com/24368665/157450921-aaa3db70-199d-4f9b-bdee-75c1c0c58568.png)

Please create a ticket for this too, thanks.  @giladt

**Comment by curvelabs-dev** [comment]
Moving this back to QA, with tickets created for the reporting issues.
